File: GitHubLike/Modules/OrganizationModule/Models/OrganizationProjectsViewDto.cs
using GitHubLike.Modules.ProjectModule.Models;

namespace GitHubLike.Modules.OrganizationModule.Models
{
    public class OrganizationProjectsViewDto
    {
        public long ProjectId { get; set; }

        public string ProjectName { get; set; }

        public string CreatedBy { get; set; }

        public DateOnly CreatedOn { get; set; }

        public List<ProjectUserDetailViewDto> OrganizationProjectUsers { get; set; }
    }
}

File: GitHubLike/frontend/src/models/organizations/OrganizationProjectsViewDto.ts
export interface OrganizationProjectsViewDto {
  projectId: number;
  projectName: string;
  createdBy: string;
  createdOn: Date;
  organizationProjectUsers: OrganizationProjectsUserDetailViewDto[];
  orgProjectUsers?: string;
}

export interface OrganizationProjectsUserDetailViewDto {
  userId: number;
  userName: string;
}
